src/styles_base.css

```
@@ -2363,7 +2363,7 @@ body.x-ray-active #echo-layer {
   left: 10%;
   width: 80%;
   height: 80%;
-  border-radius: 8px;
+  border-radius: 12px;
```

Stylelint (17.14.0)

[error] Duplicate property "border-radius" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)